.eslintrc.json

```diff
@@ -28,34 +28,34 @@
           // Similar to above but for src/www/model, but you can use files from both the
           // src/www/model and src/www/infrastructure paths.
           {
-            "target": "./src/www/model",
+            "target": "./client/src/www/model",
             "from": ".",
-            "except": ["./src/www/model", "./src/infrastructure", "./node_modules"]
+            "except": ["./client/src/www/model", "./src/infrastructure", "./node_modules"]
```

Comment:
Should `src/infrastructure` be `./infrastructure` or `./client/infrastructure`? We might need to update the `target` entry above as well.

Reply:
Good point. Updated to `./infrastructure`. I think it's okay to leave off `.client/infrastructure` for now since we intend to remove it completely. Actually, I may just put up a PR to do that now.

Reply:
#2020